media-collections/README.md

```diff
@@ -9,8 +9,19 @@ Media collections provide a simple and dynamic way to create logical groupings o
 
 You can easily add or remove assets from media collections and set their permission level for users as per your organization's needs. By doing this, you can manage your asset's workflow and access without having to delete or move it, which might break the asset's URL.
 
-Media collections also provide a convenient way for users to collaborate with each other, irrespective of the assets' actual location. It makes it easier for the administrator to manage assets, share them with different users or user groups, and manage their permissions.
+Media collections also provide a convenient way for users to collaborate with each other, irrespective of the assets' actual location. It makes it easier for the administrator to manage assets, share them with different users or user groups, and manage their permissions. For example, you can create a media collection of the latest brand assets that may be used by different teams across the organisation to ensure all communications use the latest brand assets that are managed and kept up-to-date centrally by a marketing team.
+
+ImageKit provides the following actions on media collections:
+
+1. [Create new media collection](./create-rename-delete-search-and-download-as-zip.md#create-new-media-collection)
+2. [Rename a media collection](./create-rename-delete-search-and-download-as-zip.md#rename-media-collection)
+3. [Delete media collections](./create-rename-delete-search-and-download-as-zip.md#delete-media-collections)
+4. [Search media collections](./create-rename-delete-search-and-download-as-zip.md#media-collections-search)
+5. [Download media collections as zip](./create-rename-delete-search-and-download-as-zip.md#download-media-collections-as-zip)
+6. [Add assets (files and folders) to media collections](./add-and-remove-assets-from-media-collections.md#add-assets-to-media-collections)
+7. [Remove assets (files and folders) from media collections](./add-and-remove-assets-from-media-collections.md#remove-assets-from-media-collections)
+8. [Share media collection with your team](../collaboration-and-sharing/share-media-collection.md)
 
 ## Access control and permissions
 
-Learn about actions that can be performed on media collections and associated assets by restricted media library users [here](../access-control-and-permissions/README.md#media-collection-permission-levels).
+Learn about actions that can be performed on media collections and associated assets by restricted media library users [here](../collaboration-and-sharing/README.md#media-collection-permission-levels).
```

Comment:
Needs some info about how access doesn't cascade down here. So that it's the first thing a user understands about collections.

We can put a small sentence, something like “Manage and contribute permissions set on media collection do not cascade down to its files and folders, unless those assets are also shared with a user directly.” - Followed by the link. The Wording can be improved.